frontend/src/i18n/electronI18n.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import helpers from '../helpers';
+
+// If we're running under electron, we need to communicate any language changes.
+const ipcRenderer = helpers.isElectron() ? window.require('electron').ipcRenderer : null;
+
+function tellAppAboutLanguage(lang: string) {
+  if (ipcRenderer) {
+    ipcRenderer.send('locale', lang);
+  }
+}
+
+/** Integration with Electron app, so it can change locale information. */
+export function useElectronI18n() {
+  const { i18n } = useTranslation();
+
+  useEffect(() => {
+    i18n.on('languageChanged', tellAppAboutLanguage);
+    return () => {
+      i18n.off('languageChanged', tellAppAboutLanguage);
```

Comment:
I chose to do it explicitly when the language was changed or on load because I thought we'd also have the languages at some point added to the app menu and then we could end up in calling the language set up circularly if we have it react to the language changed.
In any case this looks cleaner and we can look at the issue I described above when we have it.